platform/darwin/src/MGLOfflinePack.mm

```diff
@@ -227,19 +227,22 @@ - (void)didReceiveMaximumAllowedMapboxTiles:(uint64_t)limit {
 @end
 
 void MBGLOfflineRegionObserver::statusChanged(mbgl::OfflineRegionStatus status) {
+    __weak MBGLOfflinePack *_pack = pack;
```

Comment:
The class is called `MGLOfflinePack`, so the build is failing currently. The convention is to name the weak variable `weakPack` to more clearly distinguish the two variables.